tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```
@@ -21,6 +21,7 @@ services:
       RECEPTORCTL_SOCKET: {{ receptor_socket_file }}
       CONTROL_PLANE_NODE_COUNT: {{ control_plane_node_count|int }}
       EXECUTION_NODE_COUNT: {{ execution_node_count|int }}
+      ADMIN_PASSWORD: {{ admin_password }}
```

You might want to do:

```suggestion
      ADMIN_PASSWORD: {{ admin_password | quote }}
```

I _think_ the way this is written it would blow up if you passed special characters.